data-cassandra/docker-compose.yml

```diff
@@ -3,4 +3,4 @@ services:
   cassandra:
     image: 'cassandra:4'
     ports:
-      - '9042'
+      - '9042:9042'
```

Comment:
Isn't it possible to use dynamic ports here?

Reply:
did not work for me - ran into issues running the tests - the client was not able to connect